README.md

```diff
@@ -36,13 +36,13 @@
 
 # Personalizar - Customize
 <ul><b><i>Personalizaciones Varias - Various customizations</i></b>
-<li><p>The file --> <code>chrome</code> --> <code>components</code> --> <code>ogx_root-personal.css</code> contain a very useful config rules to a better an easy custom of the wallpaper, height tabs, Menu button's width, etc. There are descriptions on the right of each rule to you know what you can custom.</p></li></ul>
+<li><p>The file --> <code>chrome</code> --> <code>components</code> --> <code>ogx_root-personal.css</code> contains very useful and easy to use variables, so it is easier to change the wallpaper, the height of the tabs, the width of the menu button, etc. Every variable has a description on the right, so you know what to customize without breaking the file.</p></li></ul>
 <ul><b><i>Fondo Nueva Pestaña - NewTab Wallpaper</i></b>
-   <li><p>(Option 1) Into the directory: <code>chrome</code> >> <code>images</code> >> <code>newtab</code> there is a set of wallpapers , the image that the theme will select by default is the one with the name: "wallpaper.png", you could rename the image you want with that name and the theme will select that one, you could add your own images. If you don't want a wallpaper just delete the file with that name. You could find more images in <a href="https://imgur.com/a/q8SYEUJ"><b> >>This Galery<< </b></a>. <b>A restart of the browser is necessary!!!. </b></p></li>
-<li><p>(Option 2) You could edit the file --> <code>chrome</code> --> <code>components</code> --> <code>ogx_root-personal.css</code> line <code>18</code><p></li></ul></br>
+   <li><p>(Option 1) In the directory: <code>chrome</code> >> <code>images</code> >> <code>newtab</code> there is a set of wallpapers, the image that the theme will select by default is the one with the name: "wallpaper.png", you could rename the image to something else, but you will need to change some variables. If you don't want a wallpaper just delete the file with the specified name. You can find more images here: <a href="https://imgur.com/a/q8SYEUJ"><b> >>This Gallery<< </b></a>. <b>Restarting the browser after changing the wallpaper is required. </b></p></li>
```

Comment:
I'm not sure if your change in this part is what I tried to said. after the part: "wallpaper.png".

Reply:
It is what you tried to say, it just was not that clear.

Reply:
luckily i speak spanish.

Reply:
(ES) you could rename the image you want with that name and the theme will select that one, you could add your own images.
(EN) you could rename the image to something else, but you will need to change some variables.


In this part I don't think is the same idea.

Reply:
ill change that, one moment